pkg/utils/cpu_windows.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build windows
+
+package utils
+
+import (
+	"syscall"
+	"time"
+)
+
+var (
+	lastInspectUnixNano int64
+	lastCPUUsageTime    int64
+)
+
+// GetCPUPercentage calculates CPU usage and returns percentage in float64(e.g. 2.5 means 2.5%).
+func GetCPUPercentage() float64 {
```

Comment:
how can we test it? all of our test server(ci) are running at Linux.
do you have any idea?

Reply:
@leoppro I can just test on my windows, but it seems like `GetCPUPercentage` didn't have any usage now.